Makefile

```
@@ -58,7 +58,7 @@ export EIGEN_LDFLAGS :=
 ifeq ($(wildcard /usr/include/boost/version.hpp),)
 NEED_BOOST := true
 else
-NEED_BOOST := $(shell (( $$(cat /usr/include/boost/version.hpp | grep '\#define BOOST_VERSION\>' | awk '{ print $$3 }') > 106300 )) || echo true)
+NEED_BOOST := $(shell test $$(cat /usr/include/boost/version.hpp | grep '\#define BOOST_VERSION\>' | awk '{ print $$3 }') -ge 106300 || echo true)
```

Simplified to use only `awk`:
```suggestion
NEED_BOOST := $(shell awk '/\#define BOOST_VERSION\>/ { print ($$3 > 106300) ? "false" : "true" }' /usr/include/boost/version.hpp )
```

---

Actually, re-reading the Alpaka documentation, it looks like the minimum version of Boost they require is 1.65.1:
Simplified to use only `awk`:
```suggestion
NEED_BOOST := $(shell awk '/\#define BOOST_VERSION\>/ { if ($$3 < 106501) print "true" }' /usr/include/boost/version.hpp )
```